modules/admission-plug-ins-about.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-plug-ins.adoc
+
+[id="admission-plug-ins-about_{context}"]
+= About admission plug-ins
+
+Admission plug-ins are used to help regulate how {product-title} {product-version} functions. Admission plug-ins intercept requests to the master API to validate resource requests and ensure policies are adhered to. For example, they are commonly used to enforce security policy, resource limitations or configuration requirements.
```

Comment:
Admission plug-ins intercept requests to the master API to validate resource requests and ensure policies are adhered to, **after the request is authenticated and authorized.**
The is required for request.

Reply:
Resolved.